Ошибка: HTTPConnectionPool(host='go.th', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x00000152727DC6A0>: Failed to resolve 'go.th' ([Errno 11001] getaddrinfo failed)"))